app/Traits/HasJsonModelAttributes.php

```diff
@@ -63,7 +63,7 @@ public function __get($key)
         if (array_key_exists($key, $this->jsonModelAttributeCache)) {
```

Comment:
Mostly catching up with Prettier, but line 293 we bring in the rule that a deleted attribute can be dirty